documentation/SOFTWARE_REQUIREMENTS_SPEC.md

```diff
@@ -53,8 +53,8 @@ Our web-based Sokoban game will have greater user reachability compared to a mor
 The game shall be run on any browser that supports WASM (Safari, Chrome, Firefox, Edge, Opera, IOS Safari, Firefox for Android)
 
 ### Use Cases
-* The user presses the start button to initiate the game. A prompt will display asking for the user to enter level. A static world elements will be displayed on the screen.
+* The user presses the start button to initiate the game. A prompt will display asking for the user to enter level. A static board elements will be displayed on the screen.
```

Comment:
```suggestion
* The user presses the start button to initiate the game. A prompt will display asking for the user to enter level. A static board element will be displayed on the screen.
```

"Prompt" in web parlance usually means one of those annoying popup dialogs, e.g. `prompt("choose a level")`. Maybe a separate screen showing the levels graphicallly, or a modal, would be better?